### ISSUE TITLE
[Feature] Support RoadRunner HttpWorker

### ISSUE THREAD

**Issue by adrien-f** [DataDog/dd-trace-php#1858]
Greetings!

**Is your feature request related to a problem? Please describe.**

When using Roadrunner and [baldinof/roadrunner-bundle](https://github.com/baldinof/roadrunner-bundle), the current integration is not loaded because it's made to hook into `Spiral\RoadRunner\Http\PSR7Worker` [see here](https://github.com/DataDog/dd-trace-php/blob/master/src/Integrations/Integrations/Roadrunner/RoadrunnerIntegration.php#L47) while the bundle uses `Spiral\RoadRunner\Http\HttpWorker` [see here](https://github.com/Baldinof/roadrunner-bundle/blob/2.x/src/RoadRunnerBridge/HttpFoundationWorker.php#L7)

The reasons for using the HttpWorker directly appears to be for simplification and performance: https://github.com/Baldinof/roadrunner-bundle/issues/29#issuecomment-790426181

Moreover, the PSR7Worker actually wraps on the HttpWorker: https://github.com/spiral/roadrunner-http/blob/master/src/PSR7Worker.php#L102

**Describe the solution you'd like**

We would like the Roadrunner integration to support the HttpWorker

**Describe alternatives you've considered**

The other Symfony package is https://github.com/php-runtime/roadrunner-symfony-nyholm which will use the Psr7 worker, but the Baldinof/roadrunner-bundle brings a lot of extra features we could use, such as Temporal (which I heard you might be a big user) and some other features of Roadrunner.

**Additional context**

I'd be open to contributing to the change if we can find a way forward 🙏  Thank you in advance!

**Comment by bwoebi** [comment]
You are right :-) My bad, I forgot to update the autoloading for integrations.

**Comment by bwoebi** [comment]
@adrien-f I'll not immediately release a new version, but in the meantime, you can just manually explicitly load the integration via `\DDTrace\Integrations\Roadrunner\RoadrunnerIntegration::init();`.
